pkgs/by-name/br/bruno/package.nix

```
@@ -67,6 +68,10 @@ buildNpmPackage rec {
     # disable telemetry
     substituteInPlace packages/bruno-app/src/providers/App/index.js \
       --replace-fail "useTelemetry({ version });" ""
+
+    # fix version reported in about page
+    ${jq}/bin/jq '.version |= "${version}"' packages/bruno-electron/package.json > bruno-electron-package.json
+    mv bruno-electron-package.json packages/bruno-electron/package.json
```

Nit: put dependencies in `nativeBuildInputs` instead, and use `sponge` (available in `moreutils` package) to edit the file "in place".  However having one less build time dependency is also a positive, so maybe keep it as it, it's your choice :)
```suggestion
    jq '.version |= "${version}"' packages/bruno-electron/package.json | sponge packages/bruno-electron/package.json
```